.github/workflows/build_test.yaml

```diff
@@ -0,0 +1,215 @@
+name: Build and test
```

Comment:
I guess it can be named like that, but the idea is to reflect the file name, and to differentiate it with other workflows (that were not pushed when you did the review).

Reply:
If this is not the only one in workflows, your name is fine. Would you like a badge for the new workflow on README?

Reply:
For me "Build and test" is fine as a badge name, but if you want we can name it "CI" or "Continuous integration."

Reply:
I prefer "CI" then.

Reply:
Ok.